plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -1397,6 +1404,47 @@ public void setColumnComment(ConnectorSession session, ConnectorTableHandle tabl
         return viewToken;
     }
 
+    @Override
+    public Optional<CatalogSchemaTableName> redirectTable(ConnectorSession session, SchemaTableName tableName)
+    {
+        return tableRedirectionProvider.redirectTable(session, tableName);
+    }
+
+    @Override
+    public Stream<TableColumnsMetadata> streamTableColumns(ConnectorSession session, SchemaTablePrefix prefix)
```

Comment:
We should unimplement `listTableColumns` now. (https://github.com/trinodb/trino/pull/11599)